docs/software-engineering-policies/SecureDevelopment/SecurityChampion.md

```diff
@@ -0,0 +1,71 @@
+# Security Champion
```

Comment:
Umm, this isn't related to the PR topic 😮

Reply:
True.  It's because I wanted to talk about our security champion group in the readme, so I dragged it in.  A mistake, in hindsight.  What should I do?

Reply:
The risk for docs is minimal, lets not worry for now.